### DISCUSSION TITLE
Is Python v 3.11.4 compatibility on the roadmap?

### DISCUSSION THREAD

**Discussion by MaxL777** [pycaret/pycaret#3751] [General]
We have 3.11.4 installed on several machines and I'm curious if there was a plan for PyCaret to have 3.11.4 compatibility. If so, when? Thanks

**Comment by tvdboom** [comment]
We are working on it. There is a PR (#3756) but the test suite is still failing. It should be done for next release

**Comment by celestinoxp** [comment]
@MaxL777 
Information update: Pycaret 3.2.0 now supports Python 3.11.
Changelog: [https://github.com/pycaret/pycaret/releases/tag/3.2.0](https://github.com/pycaret/pycaret/releases/tag/3.2.0)

Please update pycaret to the latest version and if you encounter any problems, you can open a new issue....
